benchmark/bm_entangling_layers.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Entangling layers benchmark.
+"""
+# pylint: disable=invalid-name
+import numpy as np
+
+import pennylane as qml
+from pennylane.templates.layers import StronglyEntanglingLayers
+from pennylane.templates.embeddings import AngleEmbedding
+from pennylane.init import strong_ent_layers_uniform
+import benchmark_utils as bu
+
+
+
+def circuit(weights, *, features=None):
+    """Immutable quantum circuit."""
+
+    n_wires = len(features)  # normally not allowed in immutable circuits, but here we know the len will not change...
+    AngleEmbedding(features, range(n_wires))
```

Comment:
(just for consistency)
```suggestion
    AngleEmbedding(features, wires=range(n_wires))
```